infrastructure/elastic_resources/dashboards/ai_ssl_cert_chain_failures.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
In all other cases we have had dashboard in the file name e.g. infrastructure/elastic_resources/dashboards/ai_ssl_cert_chain_failures.**dashboard**.json. It would be good to stick to the convention